src/components/Modals/Onboarding/OnboardingTransfer.vue

```diff
@@ -192,6 +192,20 @@ type Data = {
 
 const scoresAP = [1, 2, 3, 4, 5];
 const scoresIB = [1, 2, 3, 4, 5, 6, 7];
+const existingAP: Record<string, boolean> = {};
```

Comment:
Do you think making `existingAP` and `existingIB` arrays could simplify the filtering? So you would check whether the name of the exam is in the array. Is there a benefit to using `Record` here?

Reply:
time complexity